lang/mozjs78/Portfile

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           muniversal 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+name                mozjs78
+version             78.11.0
+categories          lang
+platforms           darwin
+license             {MPL-2 LGPL-2.1+}
+maintainers         {devans @dbevans} openmaintainer
```

Comment:
Has Dave agreed to maintain this port?

Reply:
No, sorry. This is a mistake due to a copy from the mozjs60 Portfile. What is Dave's opinion about policykit he maintains and about the new mozjs78 port?